2022-03-29 09:53:04,054 INFO    MainThread:298968 [wandb_setup.py:_flush():69] setting env: {}
2022-03-29 09:53:04,054 INFO    MainThread:298968 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-29 09:53:04,054 INFO    MainThread:298968 [wandb_init.py:_log_setup():338] Logging user logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_095304-6kbk9uoc/logs/debug.log
2022-03-29 09:53:04,055 INFO    MainThread:298968 [wandb_init.py:_log_setup():339] Logging internal logs to /lila/home/harrigan/damuta/docs/examples/wandb/run-20220329_095304-6kbk9uoc/logs/debug-internal.log
2022-03-29 09:53:04,056 INFO    MainThread:298968 [wandb_init.py:_jupyter_setup():289] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f748d461850>
2022-03-29 09:53:04,056 INFO    MainThread:298968 [wandb_init.py:init():371] calling init triggers
2022-03-29 09:53:04,057 INFO    MainThread:298968 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-29 09:53:04,057 INFO    MainThread:298968 [wandb_init.py:init():420] starting backend
2022-03-29 09:53:04,057 INFO    MainThread:298968 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-29 09:53:04,082 INFO    MainThread:298968 [backend.py:ensure_launched():135] starting backend process...
2022-03-29 09:53:04,098 INFO    MainThread:298968 [backend.py:ensure_launched():139] started backend process with pid: 299251
2022-03-29 09:53:04,101 INFO    MainThread:298968 [wandb_init.py:init():425] backend started and connected
2022-03-29 09:53:04,113 INFO    MainThread:298968 [wandb_run.py:_label_probe_notebook():846] probe notebook
2022-03-29 09:53:04,114 INFO    MainThread:298968 [wandb_run.py:_label_probe_notebook():853] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-29 09:53:04,114 INFO    MainThread:298968 [wandb_init.py:init():473] updated telemetry
2022-03-29 09:53:04,115 INFO    MainThread:298968 [wandb_init.py:init():492] communicating current version
2022-03-29 09:53:04,824 INFO    MainThread:298968 [wandb_init.py:init():497] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-29 09:53:04,826 INFO    MainThread:298968 [wandb_init.py:init():505] communicating run to backend with 30 second timeout
2022-03-29 09:53:05,038 INFO    MainThread:298968 [wandb_init.py:init():530] starting run threads in backend
2022-03-29 09:53:05,923 INFO    MainThread:298968 [wandb_run.py:_console_start():1625] atexit reg
2022-03-29 09:53:05,924 INFO    MainThread:298968 [wandb_run.py:_redirect():1499] redirect: SettingsConsole.WRAP
2022-03-29 09:53:05,925 INFO    MainThread:298968 [wandb_run.py:_redirect():1536] Wrapping output streams.
2022-03-29 09:53:06,277 INFO    MainThread:298968 [wandb_run.py:_redirect():1560] Redirects installed.
2022-03-29 09:53:06,277 INFO    MainThread:298968 [wandb_init.py:init():555] run started, returning control to user process
2022-03-29 09:53:06,280 INFO    MainThread:298968 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:53:06,280 INFO    MainThread:298968 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:53:06,481 INFO    MainThread:298968 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:53:06,740 INFO    MainThread:298968 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:53:06,740 INFO    MainThread:298968 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:53:07,183 INFO    MainThread:298968 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:53:07,195 INFO    MainThread:298968 [wandb_run.py:_config_callback():874] config_cb None None {'opt_method': 'ADVI', 'seed': 2021, 'n_sigs': 20, 'alpha_bias': 0.1, 'psi_bias': 0.01, 'n': 1000, 'init_strategy': 'uniform'}
2022-03-29 09:58:45,249 INFO    MainThread:298968 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:58:45,253 INFO    MainThread:298968 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 09:58:49,325 INFO    MainThread:298968 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 09:58:49,480 INFO    MainThread:298968 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 09:58:49,480 INFO    MainThread:298968 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2022-03-29 10:02:21,756 INFO    MainThread:298968 [wandb_init.py:_resume_backend():260] resuming backend
2022-03-29 10:02:21,798 INFO    MainThread:298968 [wandb_init.py:_pause_backend():251] pausing backend
2022-03-29 10:02:21,799 INFO    MainThread:298968 [jupyter.py:save_ipynb():284] not saving jupyter notebook
